modules/swagger-codegen-maven-plugin/README.md

```diff
@@ -45,7 +45,8 @@ mvn clean compile
 - `modelPackage` - the package to use for generated model objects/classes
 - `apiPackage` - the package to use for generated api objects/classes
 - `invokerPackage` - the package to use for the generated invoker objects
-- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums.
+- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums
+- `useJaxbAnnotations` - enable Jaxb annotations inside the generated models
```

Comment:
@wing328 That is correct. This feature is only for `jaxrs-cxf`. I did not add the maven extension myself, but I am more than happy to change the vernacular to `enableJaxbAnnotations`. Is that something you would like me to do to get this PR merged?